modules/olm-installing-specific-version-cli.adoc

```diff
@@ -123,6 +123,21 @@ You can format an Operator's version and channel information in the YAML format
 $ oc get packagemanifests <operator_name> -n <catalog_namespace> -o yaml
 ----
 ====
++
+Running the `oc get packagemanifest` or `oc describe packagemanifest` commands can return a random package manifest if the following conditions are met:
```

Comment:
I would maybe rephrase

`return a random package manifest`

to

`return a package from a nondeterministic catalog source` ?

It's not that any random package will show up, it's just that the package may be from a source you may not expect if there's more than one package with the same name. Generally this means that you're going to sometimes get the community version of an operator instead of the certified one, or vice versa.